src/features/uya-online/UYAOnline.tsx

```diff
@@ -0,0 +1,135 @@
+import React, { useEffect, useState, Dispatch} from 'react';
+import { Container, Grid, Card, CardContent, Typography, CircularProgress } from '@mui/material';
+import axios from 'axios';
+import {useAppDispatch} from "../../app/hooks";
+
+import {AxiosResponse} from "axios";
+import {getHandler} from "../../utils/Requests";
+
+import {
+    UYAOnlinePlayer,
+    UYAOnlinePlayerResponse,
+    UYAOnlineGame,
+    UYAOnlineGameResponse,
+} from "../../utils/Interfaces";
+import {AnyAction} from "@reduxjs/toolkit";
+
+
+
+const UYAOnline: React.FC = () => {
+    const [players, setPlayers] = useState<UYAOnlinePlayer[]>([]);
+    const [games, setGames] = useState<UYAOnlineGame[]>([]);
+    const [loading, setLoading] = useState(true);
+    const [lastUpdated, setLastUpdated] = useState<string | null>(null); // State to track last updated timestamp
+
+    const dispatch: Dispatch<AnyAction> = useAppDispatch();
+
+
+    useEffect(() => {
```

Comment:
useEffect doesn't have a dependency array. It's generally a good rule of thumb to have each async request in it's own useEffect hook.